### PR TITLE
allow login via environment variables

### PR DESCRIPTION
**Pull request by boneskull**
This allows use of `CMD_EMAIL` and `CMD_PASSWORD` when running `codimd-cli login`.

This is needed for running `codimd-cli` in a spawned process; e.g., in a cron job.

Ref: nodejs/create-node-meeting-artifacts#72

### CODE REVIEW

**Comment by bnb** [comment]
This would help dramatically improve the Node.js project's ~dozens of monthly meetings. Would genuinely love to see this shipped.

**Comment by Yukaii** [comment]
Hi @boneskull There're some conflicts since #4 is merged.

I checked the ref and found you mentioned hackmd.io in the comment: https://github.com/nodejs/create-node-meeting-artifacts/issues/72#issuecomment-596779310. Currently `codimd-cli` works only with [CodiMD](https://github.com/hackmdio/codimd) and is not compatible with HackMD.io. (Only Need to deal with some CSRF things)

**Comment by boneskull** [comment]
@Yukaii Thanks.  Jong-Kai reached out about this, so I'll be following up there.  We'll be determining  whether the Node.js project needs its own CodiMD instance, or if HackMD.io can support our use case.

**Comment by boneskull** [comment]
I've resolved that conflict and have tested against a locally-running CodiMD instance.

File: src/commands/login.ts [hunk 1]
```diff
@@ -24,7 +25,8 @@ Login as HMD successfully!
   async run() {
     const {flags} = this.parse(Login)
 
-    let email = flags.email
+    let email = flags.email || process.env.CMD_EMAIL
```
Comment (Yukaii) [inline src/commands/login.ts]: Hi @boneskull Could you please update the environment variable with `CMD_CLI` prefix? I want  it different from [CodiMD's configuration](https://hackmd.io/c/codimd-documentation/%2Fs%2Fcodimd-configuration#Email).

File: README.md [hunk 2]
```diff
@@ -171,6 +171,8 @@ EXAMPLE
   Login as HMD successfully!
 ```
 
+Alternatively, you may supply an email and password via the `CMD_EMAIL` and `CMD_PASSWORD` environment variables, respectively.
```
Comment (Yukaii) [inline README.md]: There's a configuration table in the above section. Could you please also add these two keys to it?

**Comment by Yukaii** [comment]
@boneskull hackmd.io support is done in #9, could you rebase your work against the latest master branch?

**Comment by Yukaii** [comment]
@boneskull Reorder as follows would fix this

image

**Comment by boneskull** [comment]
I've updated the env vars to have `CMD_CLI_` prefix, and updated `README.md`

**Comment by Yukaii** [comment]
Thank you @boneskull !

**Pull request merged**